app/Repositories/PhotoRepository.php

```diff
@@ -44,7 +44,11 @@ public function create(array $data, $signupId)
     {
         $postEvent = Event::create($data);
 
-        $fileUrl = $this->aws->storeImage($data['file'], $data['campaign_id']);
+        if (array_key_exists('campaign_id', $data)) {
+            $fileUrl = $this->aws->storeImage($data['file'], $data['campaign_id']);
+        } else {
+            $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+        }
```

Comment:
@sbsmith86 yeah, I think `signupId` makes sense in the new world, but I think we need the if/else until Phoenix is sending to the signups endpoint to not break how this works currently